src/libexpr/eval.cc

```diff
@@ -465,6 +465,24 @@ EvalState::~EvalState()
 }
 
 
+void EvalState::requireExperimentalFeatureOnEvaluation(
+    const std::string & feature,
+    const std::optional<std::string> & fName,
+    const Pos & pos)
+{
+    if (!settings.isExperimentalFeatureEnabled(feature)) {
+        auto f = fName.value_or("a function");
+        throw EvalError({
+            .msg = hintfmt(
+                "Experimental Nix feature '%1%' is missing to call %2%! You can it via '--extra-experimental-features %1%'.",
```

Comment:
There's a missing word:

```suggestion
                "Experimental Nix feature '%1%' is missing to call %2%! You can enable it via '--extra-experimental-features %1%'.",
```

An alternative wording:

```suggestion
                "Experimental Nix feature '%1%' must be enabled to use %2%! You can enable it via '--extra-experimental-features %1%'.",
```